README.md

```diff
@@ -476,7 +478,19 @@ end
 ```
 
 Once the new token is created, the old one is revoked before returning the new
-token.
+token. 
+
+### Refresh Tokens
+
+You can create refresh token, which by default are long living. These can later be exhanged for shorter living access tokens, which can be used as a authorization token.
+
+```elixir
+    #issue a long living refresh token
```

Comment:
Could you add a space after `#` and capitalize Issue?  Same feedback for the comment below.